compiler/rustc_smir/src/rustc_smir/context.rs

```diff
@@ -228,7 +228,7 @@ impl<'tcx> Context for TablesWrapper<'tcx> {
         }
     }
 
-    fn span_to_string(&self, span: stable_mir::ty::Span) -> String {
+    fn span_to_string(&self, span: stable_mir::ty::Span) -> std::borrow::Cow<'static, str> {
```

Comment:
Maybe `span_to_cow`

Reply:
same as my above -- `span_to_string` is probably appropriate name to keep.